semgrep/dependencyparser/models.py

```diff
@@ -24,7 +23,7 @@ class PackageManagers(Enum):
     MAVEN = "maven"
 
 
-@frozen(eq=True, order=True)
+@dataclass(eq=True, order=True, frozen=True)
```

Comment:
This `dataclass` change was to make JSON serialization possible without doing it manually

Reply:
Even better would be to use atdpy!
That way this can also be consumed in a type-safe way on the other side (semgrep-app backend)